src/components/VHeader/VHeaderMobile/VBackButton.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <VButton
+    size="disabled"
+    class="icon-button flex h-12 w-12 flex-shrink-0 rotate-90 items-center justify-center bg-tx active:shadow-ring"
+    variant="plain--avoid"
+    aria-label="Go back"
```

Comment:
```suggestion
    aria-label="Go back"
```

Should this be added to `en.json`?

Reply:
Definitely! Thank you for catching.